docs/accessing_transaction_field.rst

```diff
@@ -7,43 +7,60 @@ PyTeal smart contracts can access properties of the current transaction and the
-:any:`Txn.first_valid() <TxnObject.first_valid>`                                 :code:`TealType.uint64`   2                round number 
+:any:`Txn.first_valid() <TxnObject.first_valid>`                                 :code:`TealType.uint64`   2                round number
 :any:`Txn.last_valid() <TxnObject.last_valid>`                                   :code:`TealType.uint64`   2                round number
 :any:`Txn.note() <TxnObject.note>`                                               :code:`TealType.bytes`    2                transaction note in bytes
 :any:`Txn.lease() <TxnObject.lease>`                                             :code:`TealType.bytes`    2                transaction lease in bytes
-:any:`Txn.receiver() <TxnObject.receiver>`                                       :code:`TealType.bytes`    2                32 byte address
-:any:`Txn.amount() <TxnObject.amount>`                                           :code:`TealType.uint64`   2                in microAlgos
-:any:`Txn.close_remainder_to() <TxnObject.close_remainder_to>`                   :code:`TealType.bytes`    2                32 byte address
+:any:`Txn.group_index() <TxnObject.group_index>`                                 :code:`TealType.uint64`   2                position of this transaction within a transaction group, starting at 0
+:any:`Txn.tx_id() <TxnObject.tx_id>`                                             :code:`TealType.bytes`    2                the computed ID for this transaction, 32 bytes
+:any:`Txn.rekey_to() <TxnObject.rekey_to>`                                       :code:`TealType.bytes`    2                32 byte address
+================================================================================ ========================= ================ ============================================================================
+
+Transaction Fields:  Key Registration
+*************************************
+================================================================================ ========================= ================ ============================================================================
+Operator                                                                         Type                      Min TEAL Version Notes
+================================================================================ ========================= ================ ============================================================================
 :any:`Txn.vote_pk() <TxnObject.vote_pk>`                                         :code:`TealType.bytes`    2                32 byte address
 :any:`Txn.selection_pk() <TxnObject.selection_pk>`                               :code:`TealType.bytes`    2                32 byte address
+:any:`Txn.state_proof_pk <TxnObject.state_proof_pk>`                             :code:`TealType.bytes[]`  6                64 byte state proof public key commitment.
```

Comment:
The type for this should be `TealType.bytes`

Reply:
@jasonpaulos Thanks for catching - I fixed the data type via https://github.com/algorand/pyteal/pull/204/commits/f7616ab188e6ae2c1b2a5c9eebd1b4579520ebd5.